README.md

```diff
@@ -1,21 +1,29 @@
 # obsidian [![Tek-OSS](https://tektronix.github.io/media/TEK-opensource_badge.svg)](https://github.com/tektronix)
+
 Displays Jenkins build status with a string of RGB LED
 
+## Maintainer
+
+-   [Hong Quach](https://github.com/htquach)
+
 ## BOM
-1. A Raspberry Pi with Raspbian OS
-    * CanaKit Raspberry Pi 3 B+ (B Plus) with Premium Clear Case and 2.5A Power Supply
-    * A microSD card for the Raspberry Pi to run the Raspbian OS
-1. WS2812B addressable RGB LED strip 
-    * 2x WS2812B RGB 144 LED Strip Light 5V White PCB Waterproof IP67)
-    * 1x 5V 10A power supply to drive the LEDs
+
+1.  A Raspberry Pi with Raspbian OS
+    -   CanaKit Raspberry Pi 3 B+ (B Plus) with Premium Clear Case and 2.5A Power Supply
+    -   A microSD card for the Raspberry Pi to run the Raspbian OS
+
+2.  WS2812B addressable RGB LED strip
```

Comment:
Looks like the numbering changed.  Is this intentional?  Using all '1.' in an md file allows inserting new entry without having to shift the other numbers.